spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -1264,6 +1266,23 @@ public void testRewriteJobOrderFilesDesc() {
     Assert.assertNotEquals("Number of files order should not be ascending", actual, expected);
   }
 
+  @Test
+  public void testPartitioningSortWithManualSortOnSameColumn() {
+    Table table = createTablePartitioned(4, 2);
+    table.refresh();
+    table.updateSpec().addField(bucket("c3", 16)).commit();
+    Assert.assertFalse(table.specs().isEmpty());
+    Assert.assertFalse(table.spec().fields().isEmpty());
```

Comment:
I don't think that these assertions are needed, since this isn't trying to validate the behavior of `updateSpec`. This should just assume that it the update worked.

Reply:
I have removed it.